pkg/cmd/rebuild.go

```diff
@@ -69,7 +69,7 @@ func (o *rebuildCmdOptions) rebuild(_ *cobra.Command, args []string) error {
 		return err
 	}
 
-	fmt.Printf("%d integrations have been rebuilt\n", len(integrations))
+	fmt.Fprintf(cmd.OutOrStdout(),"%d integrations have been rebuilt\n", len(integrations))
```

Comment:
Another missing space. Likely you want to run `gofmt` to fix all formatting errors.